spec/amazon_ses_mailer/message_spec.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+RSpec.describe AmazonSesMailer::Message do
+  let(:template_name) { 'welcome_email' }
+  let(:options) { { template: template_name, reply_to: 'saba@gmail.com' } }
+  subject { described_class.new(template_name) }
+  describe '.ensure_array' do
+    context 'when string is given in option' do
+      it 'should return a string' do
```

Comment:
From the method name, it should convert the string into an array. This is also not a unit test; it should not call other methods from other classes like `Base.mail`.